tailwind.config.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  purge: {
+    mode: 'production',
+    content: ['./src/**/*.html'],
+  },
+  future: {
+    removeDeprecatedGapUtilities: true,
+  },
+  theme: {
+    extend: {},
+    colors: {
+      // this colors are copied from Yeti
+      transparent: 'transparent',
+      blue: '#008cba',
+      indigo: '#6610f2',
+      purple: '#6f42c1',
+      pink: '#e83e8c',
+      red: '#F04124',
+      orange: '#fd7e14',
+      yellow: '#E99002',
+      green: '#43ac6a',
+      teal: '#20c997',
+      cyan: '#5bc0de',
+      white: '#fff',
+      gray: '#888',
+      'gray-dark': '#333',
+      primary: '#008cba',
+      secondary: '#eee',
+      success: '#43ac6a',
+      info: '#5bc0d',
+      warning: '#E99002',
+      danger: '#F04124',
+      light: '#eee',
+      dark: '#222',
+    },
+    screens: {
+      // this breakpoints are copied from Yeti
+      'xs': '0',
```

Comment:
I might want to remove this breakpoint, since it is not very efficient to generate css for a breakpoint like this XD